go/vt/sqlparser/ast.go

```diff
@@ -2065,6 +2071,12 @@ func (node *TableIdent) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
+// Lowered returns a new TableIdent where the backing string has been
+// converted to lowercase
+func (node TableIdent) Lowered() TableIdent {
```

Comment:
Since this function should generally not be used from outside this package, you could make it private, or even get rid of it and do the conversion directly in `ToViewName`.